index.html

```diff
@@ -1,28 +1,118 @@
 <!DOCTYPE html>
 <html lang="en">
-
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="stylesheet" href="style.css">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <link
+      href="https://unpkg.com/boxicons@2.1.4/css/boxicons.min.css"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="style.css" />
     <title>Tracom Academy website</title>
-</head>
-
-<body>
+  </head>
+  <body>
     <header class="main-header">
-        <nav class="main-nav">
-            <img src="Assets/tracom_services_cover.jpg" alt="tracom-logo" class="main-logo">
-            <ul id="main-nav-ul">
-                <li><a href="#">Home</a></li>
-                <li><a href="#">About</a></li>
-                <li><a href="#">Services</a></li>
-                <li><a href="#">Team</a></li>
-                <li><a href="#">Contact</a></li>
-            </ul>
-        </nav>
+      <nav class="main-nav">
+        <img
+          src="Assets/tracom_services_cover.jpg"
+          alt="tracom-logo"
+          class="main-logo"
+        />
+        <ul id="main-nav-ul">
+          <li><a href="#">Home</a></li>
+          <li><a href="#">About Us </a></li>
+          <li><a href="#">Services</a></li>
+          <li><a href="#">Team</a></li>
+          <li><a href="#">Contact</a></li>
```

Comment:
link your contact page here

Reply:
make your page dynamic such that one can switch from landing page to contact page using the navigation links